department-of-veterans-affairs/vets-api
# BE | `users/dashboard` endpoint

**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.


## Summary

- created a DynamicsService to do Api calls to Dynamics
- created a Datadog logger to log responses
- created refactored the Dashboard flow and out of PoC
- updated other classes to pass tests
- restructure ask_va_api

Issue #238

## Related issue(s)
- department-of-veterans-affairs/ask-va#283

## Testing done

- All functionality in `vets-website` for `ask_va` should still work as it is
- Need to check datadog in staging
  - make sure that we can see the log
- Need to check sentry in staging
  - make sure that we can see the log

## Screenshots
_Note: Optional_


## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria


## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
